app.js

```diff
@@ -38,4 +38,17 @@ app.locals.title = 'Express - Generated with IronGenerator';
 const index = require('./routes/index');
 app.use('/', index);
 
+const movies = require('./routes/movies.js');
+app.use('/movies', movies);
+
+// SHORT SOLUTION
+// const Movie = require("./models/Movie.model")
+// console.log(Movie)
+// app.get("/movies", (req,res) => {
+//   Movie.find().then(allMovies => {
+//       console.log(allMovies)
+//       res.render('movies', {movies: allMovies})
+//   })
+// })
```

Comment:
Good to see this short solution, would definitely work perfectly for showing all movies! However creating different route files for separating groups of functionalities is always the preferred way. Particularly in big projects It will make the code more readable and easier to collaborate on :)